src/pages/workspaces/workspace/workflows/FailureRerunner.js

```diff
@@ -27,7 +27,7 @@ export const rerunFailures = async ({ workspace, workspace: { workspace: { names
   const eventData = { ...extractWorkspaceDetails(workspace), configNamespace, configName }
 
   try {
-    const [{ workflows, useCallCache, deleteIntermediateOutputFiles, useReferenceDisks }, config] = await Promise.all([
+    const [{ workflows, useCallCache, deleteIntermediateOutputFiles, useReferenceDisks, memoryRetryMultiplier }, config] = await Promise.all([
```

Comment:
To test that this works, I had to hack the JS to show the "Relaunch Failures" Action on the Job History page. It is not enabled for the test WDL I am running.